src/Symfony/Component/Security/Http/Authenticator/BodyAccessTokenAuthenticator.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http\Authenticator;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Core\User\UserProviderInterface;
+use Symfony\Component\Security\Http\Authentication\AuthenticationFailureHandlerInterface;
+use Symfony\Component\Security\Http\Authentication\AuthenticationSuccessHandlerInterface;
+
+/**
+ * Provides an implementation of the RFC6750 of an authentication via
+ * an access token in the request Body.
+ *
+ * @author Florent Morselli <florent.morselli@spomky-labs.com>
+ *
+ * @final
+ */
+class BodyAccessTokenAuthenticator extends AbstractAccessTokenAuthenticator
```

Comment:
I would suggest 1 unique authenticator, and use an extractor to retrieve the token following the standard: `Authorization` header, `access_token` body, `accessToken` query

Reply:
At first sight I wanted to have only one authenticator.
I will try to merge them.

Reply:
Done in the last commits